include/aws/auth/signable.h

```diff
@@ -215,7 +215,7 @@ AWS_AUTH_API
 struct aws_signable *aws_signable_new_trailing_headers(
     struct aws_allocator *allocator,
     struct aws_http_headers *trailing_headers,
-    struct aws_byte_cursor previous_signature);
+    const struct aws_byte_cursor previous_signature);
```

Comment:
```suggestion
    struct aws_byte_cursor previous_signature);
```
the byte cursor is already passed by value/copy
there's no point in making it const